src/zos.cc

```diff
@@ -26,6 +25,7 @@
 #include <dlfcn.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <features.h>
```

Comment:
Since you included other system headers you don't need to include this one directly.